mm/ktsan/stack.c

```diff
@@ -14,7 +14,6 @@ void kt_stack_print(kt_stack_t *stack, uptr_t top_pc)
 		pc = kt_decompress(stack->pc[i]);
 		pr_err(" [<%p>] %pS\n", (void *)pc, (void *)pc);
 	}
-	pr_err("\n");
```

Comment:
But in that case reports look like this:
```
==================================================================
ThreadSanitizer: data-race in rcu_read_under_lock

Read at 0xffff88028a19b400 of size 4 by thread 3052 on CPU 1:
 [<ffffffff81252e66>] rcu_read_under_lock+0x16/0x30 mm/ktsan/tests_inst.c:510
 [<ffffffff812523fd>] thr_func+0x2d/0x50 mm/ktsan/tests_inst.c:33
 [<ffffffff810b98c1>] kthread+0x161/0x180 kernel/kthread.c:209
 [<ffffffff81ea8c8f>] ret_from_fork+0x3f/0x70 arch/x86/entry/entry_64.S:529

Previous write at 0xffff88028a19b400 of size 4 by thread 3053 on CPU 2:
 [<ffffffff81252e36>] rcu_write_under_lock+0x16/0x30 mm/ktsan/tests_inst.c:523
 [<ffffffff812523fd>] thr_func+0x2d/0x50 mm/ktsan/tests_inst.c:33
 [<ffffffff810b98c1>] kthread+0x161/0x180 kernel/kthread.c:209
 [<ffffffff81ea8c8f>] ret_from_fork+0x3f/0x70 arch/x86/entry/entry_64.S:529

==================================================================
```

And that trailing newline before ====== doesn't look good, imho.

All other kt_stack_print are followed by pr_err("\n") in report.c